test/integration/run-command.js

```diff
@@ -30,11 +33,37 @@ function runCommand(path, args, options) {
         return;
       }
       if (code !== 0) {
-        reject(new Error('exit code: ' + code));
+        reject(new Error(
+          `Error running ${path} with args ${args}. Got exit code: ${code}`));
         return;
       }
       resolve();
     });
+  }).catch((err) => {
+    return contents.then((out) => {
```

Comment:
it took a while to figure out what you're trying to do here, a brief comment help a lot